modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/schema/TableDescriptor.java

```diff
@@ -109,4 +110,11 @@ default RelDataType selectForUpdateRowType(IgniteTypeFactory factory) {
      * @return Actual count of columns.
      */
     int columnsCount();
+
+    /**
+     * Returns list of colocation columns of the table.
+     *
+     * @return Actual list of colocation columns.
+     */
+    List<Integer> colocationColumns();
```

Comment:
why do we need this?

Reply:
Removed from the interface, left only in the implementation.

Reply:
Why don't just use distribution.getKeys() instead?

Reply:
Fixed, thanks.